johnsonandjohnson/Bodiless-JS
# feat(backend, components): Image files should be uploaded to path per page

## Changes
- Images are uploaded to a static folder path that is unique to the page and hash of image `nodePath`.
- New Cypress helper `isImageVisible(imageXpath)` that can be used to test if the image was successfully loaded and its `src` is not 404.
## Test Instructions
- Upload images from different pages and confirm image path matching `/images/pages/{pagepath}/{hash}/{filename}`
- Images uploaded from the home page should be saved to `/images/site/{hash}/{filename}`

## Related Issues
Closes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/441
